java/src/test/java/ai/rapids/cudf/RmmTest.java

```diff
@@ -399,6 +399,14 @@ public void testPoolLimitNonPoolMode() {
         () -> Rmm.initialize(RmmAllocationMode.CUDA_DEFAULT, false, 1024, 2048));
   }
 
+  @Test
+  public void testGetCudaRuntimeInfo() {
+    Rmm.initialize(RmmAllocationMode.POOL, false, 1024);
```

Comment:
What happens if we run these APIs without initializing the pool?  Because in the common case I suspect that is how they are going to be used.

Also why are the tests a part of RMM.  They should have nothing to do with RMM.

Reply:
I guess this initialization is used to set the GPU device for the later operations, per the current JNI implementation.  But even so, moving these tests to Cuda would be better.

And I am wondering whether all the calls in CudaJni need to set the device first.

Reply:
I moved this case to a newly-created test class `CudaTest`

Reply:
And there is no necessary to initialize RMM before running these APIs.